src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockCase.java

```diff
@@ -13,13 +13,13 @@
     void method1(int a) {
         switch (a) {}
         switch (a) {case 1: ; } // ok
-        switch (a) {case 1:{}}  // violation
+        switch (a) {case 1:{}}  // violation ,Empty block
```

Comment:
remove ','
format it like:
https://github.com/checkstyle/checkstyle/blob/5b2a0efc8c577d6382effe818b0d94d489397352/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/declarationorder/InputDeclarationOrderOnlyModifiers.java#L16